manifests/params.pp

```diff
@@ -97,15 +97,7 @@
         $python_package_name = pick($python_package_name, 'python-psycopg2')
       }
 
-      if $postgresql::globals::postgis_package_name {
-        $postgis_package_name = $postgresql::globals::postgis_package_name
-      } elsif $facts['os']['release']['major'] == '5' {
-        $postgis_package_name = 'postgis'
-      } elsif $postgis_version and versioncmp($postgis_version, '2') < 0 {
-        $postgis_package_name = "postgis${package_version}"
-      } else {
-        $postgis_package_name = "postgis2_${package_version}"
-      }
+      $postgis_package_name = pick($postgresql::globals::postgis_package_name, "postgis2_${package_version}")
```

Comment:
I think this also requires its own PR. If I look at EL8 & EL9 then I don't find postgis packages. In Fedora it's just called `postgis`. No idea what it's called in the official postgresql repositories.